src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -40,6 +40,8 @@
                 ],
     'Imper': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0,         # Impervious surface area percentage
               0.15, 0.52, 0.87, 0.15, 0.52, 0.87],  # only defined for urban land use types
+    'C': [0.03, 0.42, 0.002, 0.01, 0.1, 0.03, 0.04, 0.001, 0.01, 0.08,  # C Factor Defaults
```

Comment:
I think the 5th value (disturbed) and 10th (unpaved) might be off.  If the comment below is accurate, then disturbed should should be 0.08 and unpaved should be 0.8.
```
        If Lu = 12 Or Lu = 13 Or Lu = 11 Or Lu = 15 Then ' Disturbed, Bare Rock, Unpaved Roads, & Open Land
            C12 = 0.08 : C13 = 0.1 : C11 = 0.8 : C15 = 0.08
```